src/enums/filingTypes.ts

```diff
@@ -11,7 +11,7 @@ export enum FilingTypes {
   DISSOLUTION = 'dissolution',
   DISSOLVED = 'dissolved',
   INCORPORATION_APPLICATION = 'incorporationApplication',
-  INVOLUNTARY_DISSOLUTION = 'Involuntary Dissolution',
+  INVOLUNTARY_DISSOLUTION = 'involuntaryDissolution',
```

Comment:
This looks like a past global-search-and-replace error. AFAIK, all filing types are snakeCase.

Reply:
Update: this type has not been added to the schema yet but I'm confident it will follow convention.

Ref: https://github.com/bcgov/business-schemas/blob/e32e6dc561abebdc3034af48993da12f882dfb80/src/registry_schemas/schemas/filing.json#L47